gf-guide/onboarder-workflow.md

```diff
@@ -12,7 +12,7 @@
 This means several things:
 
 <ol>
-    <li> While in the past spontaneous contributions were still possible, today the specifics of the requirements and the security in place have made it complicated for a random user to make such a contribution. As an example, the continuous integration (CI) can only get triggered if a pull request (PR) has been made by someone with permission to the repo. If a user makes a PR, it will definitely draw some attention, but the PR often needs to be re-done by a team member. So while contribution to the repo is not in theory exclusive to the “onboarders” (paid contractors), it tends to be in practice.</li>
+    <li> While in the past spontaneous contributions were still possible, today the specifics of the requirements and the security in place have made it complicated for a random user to make such a contribution. If a user makes a PR, it will definitely draw some attention, but the PR often needs to be re-done by a team member. So while contribution to the repo is not in theory exclusive to the “onboarders” (paid contractors), it tends to be in practice.</li>
```

Comment:
"spontaneous" and "random" seem overly pessimistic to me. My view of the onboarding role is they should act like a developer relations profesional, where their goal is to delegate to, educate, and empower people who want to use Google Fonts for distribution. The onboarding tools, process, and documentation can be improved to the point where known open-source font designers can contribute to the repo. That will allow Google Fonts to scale and move musc faster. When only a small group of onboarders do this work and mainly comunicate in private when important information can't be indexed by a search engin like Google, and don't create tools and workflows that can be used by contributors, thats creates a bottle neck that slows down how fast Google Fonts can move. We are going to see a LOT more high-quality open source fonts due to AI and cross-platform open source editors like fontra, so we need to take deligating work seriously in order to scale and move faster.

Reply:
You will have to see that with Dave, I didn't make that rules :)